tests/functional/_packages/cycle_type_checking/b.py

```diff
@@ -0,0 +1,7 @@
+# pylint: disable=missing-docstring
```

Comment:
Packages in functional tests are supposed to work (Matus Valo did a MR where it worked recently.) There's also a legacy functional test in test_func for cyclic import. (I thought the new functional test did not permit it).

Reply:
I believe you meant the tests here: #4678. After taking a closer look, they still only test a single module, no package.
It does however work with the old `test_func` framework.